Test failure on Windows

Test failure on Windows, but not on Mac.

```
× eject command - eject copy config - should run copy files (0s)
AssertionError: expected false to be true
  at Assertion.<anonymous>  <node_modules\intern\browser_modules\chai\chai.js:567:10>
  at Assertion.addProperty  <node_modules\intern\browser_modules\chai\chai.js:4240:29>
  at Function.assert.isTrue  <node_modules\intern\browser_modules\chai\chai.js:2483:31>
  at moduleUnderTest.run.then  <tests\unit\commands\eject.ts:143:11>
  at runMicrotasksCallback  <internal\process\next_tick.js:64:5>
  at _combinedTickCallback  <internal\process\next_tick.js:73:7>
  at process._tickCallback  <internal\process\next_tick.js:104:9> 
```

To reproduce,

```
git clone https://github.com/dojo/cli.git
cd cli
npm i
grunt test
```

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -1,0 +1,25 @@
+# Test against the latest version of this Node.js version
+environment:
+  nodejs_version: "6"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - npm install grunt cli
+  - npm install
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - node --version
+  - npm --version
+  # run tests
+  - node_modules\.bin\grunt.cmd
+  - node_modules\.bin\grunt.cmd intern:node
+  - node_modules\.bin\grunt.cmd remapIstanbul:ci
+  - node_modules\.bin\grunt.cmd uploadCoverage
+
+# Don't actually build.
+build: off

--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 # @dojo/cli
 
-[![Build Status](https://travis-ci.org/dojo/cli.svg?branch=master)](https://travis-ci.org/dojo/cli) [![codecov](https://codecov.io/gh/dojo/cli/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/cli) [![npm version](https://badge.fury.io/js/%40dojo%2Fcli.svg)](https://badge.fury.io/js/%40dojo%2Fcli)
+[![Build Status](https://travis-ci.org/dojo/cli.svg?branch=master)](https://travis-ci.org/dojo/cli) [![Build status](https://ci.appveyor.com/api/projects/status/mvbjrd0jcv8itvho/branch/master?svg=true)](https://ci.appveyor.com/project/Dojo/cli/branch/master)
+ [![codecov](https://codecov.io/gh/dojo/cli/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/cli) [![npm version](https://badge.fury.io/js/%40dojo%2Fcli.svg)](https://badge.fury.io/js/%40dojo%2Fcli)
 
 The CLI is the officially supported way to create and maintain Dojo 2 apps.
 

--- a/tests/unit/commands/eject.ts
+++ b/tests/unit/commands/eject.ts
@@ -1,7 +1,7 @@
 import { yellow, underline } from 'chalk';
 import { beforeEach, afterEach, describe, it } from 'intern!bdd';
 import * as assert from 'intern/chai!assert';
-import { join, resolve as pathResolve } from 'path';
+import { join, resolve as pathResolve, sep } from 'path';
 import * as sinon from 'sinon';
 
 import { CommandsMap, CommandWrapper } from '../../../src/command';
@@ -140,8 +140,8 @@ describe('eject command', () => {
 			const helper = getHelper();
 			mockAllExternalCommands.loadExternalCommands = sandbox.stub().resolves({commandsMap: commandMap});
 			return moduleUnderTest.run(helper, {}).then(() => {
-				assert.isTrue(consoleLogStub.secondCall.calledWith(` ${yellow('creating')} ./config/test-group-test-eject/file1`));
-				assert.isTrue(consoleLogStub.thirdCall.calledWith(` ${yellow('creating')} ./config/test-group-test-eject/file2`));
+				assert.isTrue(consoleLogStub.secondCall.calledWith(` ${yellow('creating')} .${sep}config${sep}test-group-test-eject${sep}file1`));
+				assert.isTrue(consoleLogStub.thirdCall.calledWith(` ${yellow('creating')} .${sep}config${sep}test-group-test-eject${sep}file2`));
 				assert.isTrue(mockFsExtra.copySync.calledTwice);
 			});
 		});

